ctor/src/lib.rs

```diff
@@ -253,6 +260,7 @@ pub fn ctor(_attribute: TokenStream, function: TokenStream) -> TokenStream {
 /// ```rust
 /// # extern crate ctor;
 /// # use ctor::*;
+/// # fn main() {}
```

Comment:
This is needed to prevent doctest from wrapper the whole thing in an implicit `main()` which breaks the `super::` imports. This would have technically already been broken in that defining a dtor inside another function would already have failed to import anything from the enclosing function.